community/contributing/editor_and_docs_localization.rst

```diff
@@ -0,0 +1,412 @@
+    `Have a look here <https://docs.godotengine.org/en/latest>`_.
+
+    # "|supported|" is an inline reference to an image and should stay unchanged.
+    # "master" uses the markup for inline code, and will be styled as such.
+    # Note: Inline code in RST uses 2 backticks on each side, unlike Markdown.
+    # Single backticks are used for hyperlinks.
+
+    |supported| Backwards-compatible new features (backported from the ``master``
+    branch) as well as bug, security, and platform support fixes.
+
+    # The :ref: Sphinx "role" is used for internal references to other pages of
+    # the documentation.
+    # It can be used with only the reference name of a page (which should not be
+    # changed), in which case the title of that page will be displayed:
+
+    See :ref:`doc_ways_to_contribute`.
+
+    # Or it can be used with an optional custom title, which should thus be translated:
+
+    See :ref:`how to contribute <doc_ways_to_contribute>`.
+
+    # You may encounter other Sphinx roles, such as :kbd: used for shortcut keys.
+    # You can translate the content between backticks to match the usual key names,
+    # if it's different from the English one.
+
+    Save the scene. Click Scene -> Save, or press :kbd:`Ctrl + S` on Windows/Linux
+    or :kbd:`Cmd + S` on macOS.
+
+.. seealso::
+
+    See Sphinx's `reStructured Text primer <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`__
+    for a quick overview of the markup language you may find in source strings.
+    You may encounter especially the inline markup (bold, italics, inline code)
+    and the internal and external hyperlink markup.
+
+Class reference (BBCode)
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The class reference is documented in the main Godot repository using XML files,
+and with BBCode-like markup for styling and internal references.
+
+Some of the tags used are from the original BBCode (e.g. ``[b]Bold[/b]`` and
+``[i]Italics[/i]``), while others are Godot-specific and used for advanced
+features such as inline code (e.g. ``[code]true[/code]``), linking to another
+class (e.g. ``[Node2D]``) or to a property in a given class (e.g.
+``[member Node2D.position]``), or for multiline code blocks. Example::
+
+    Returns a color according to the standardized [code]name[/code] with [code]alpha[/code] ranging from 0 to 1.
+    [codeblock]
+    red = ColorN("red", 1)
+    [/codeblock]
+    Supported color names are the same as the constants defined in [Color].
+
+In the above example, ``[code]name[/code]``, ``[code]alpha[/code]``, and
+``[Color]`` should *not* be translated, as they refer respectively to argument
+names and a class of the Godot API. Similarly, the contents of the
+``[codeblock]`` should not be translated, as ``ColorN`` is a function of the
+Godot API and ``"red"`` is one of the named colors it supports. At most, you can
+translate the name of the variable which holds the result (``red = ...``).
```

Comment:
I guess this will be useful, I've seen some nice stuff like this (since fixed):
```
Gibt eine Farbe gemäß dem standardisierten [code]Namen[/code] zurück, wobei [code]alpha[/code] im Bereich von 0 bis 1 liegt.
[codeblock]
rot = FarbeN("rot", 1)
[/codeblock]
Die unterstützten Farbnamen sind die gleichen wie die in [Farbe] definierten Konstanten.
```